src/amo/components/HeroRecommendation/index.js

```diff
@@ -0,0 +1,122 @@
+/* @flow */
+import invariant from 'invariant';
+import * as React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import type { I18nType } from 'core/types/i18n';
+
+import promoImage from './img/christin-hume-mfB1B1s4sMc-unsplash.png';
+import './styles.scss';
+
+type Props = {|
+  body: string,
+  heading: string,
+  linkText: string,
+  linkHref: string,
+|};
+
+type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+|};
+
+export class HeroRecommendationBase extends React.Component<InternalProps> {
+  renderOverlayShape() {
+    const gradientA = 'HeroRecommendation-gradient-a';
+    const gradientB = 'HeroRecommendation-gradient-b';
+
+    return (
+      <svg
+        className="HeroRecommendation-overlayShape"
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 334 307"
+      >
+        <defs>
+          <linearGradient
+            id={gradientA}
+            x1="37.554%"
+            x2="20.41%"
+            y1="51.425%"
+            y2="49.159%"
+          >
+            <stop
+              className="HeroRecommendation-gradientA-startColor"
```

Comment:
Why not reusing the `gradientA` variable to build this class name?

Reply:
I prefer not to use variables to generate css class names because if it goes out of sync with the css file, there is no warning or error. It is also harder to grep for the usage of a class name if you are reading a css file and trying to discover where it is used.